calculator2.py

```diff
@@ -0,0 +1,13 @@
+def add(x, y):
```

Comment:
Add type hints to function signatures. For example, the addition function can be defined as `def add(x: float, y: float) -> float:`. This will make the code more readable and help prevent potential type-related bugs. [important]